docs/quality-assurance/Web-Accessibility.md

```diff
@@ -0,0 +1,26 @@
+# Web Accessibility Testing
+
+Accessibility is not a 'nice to have', it's a legal requirement for companies and organisations to be inclusive for everyone.
```

Comment:
one of the problems is understanding what is and isn't required of new and existing applications. Can there be a clear section to clarify this, even if it simply contains links. I believe this would help people to focus on what is required.